src/composables/accounts.ts

```diff
@@ -11,14 +11,18 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
   /**
    * Accounts data formatted as the form select options
    */
-  const accountsSelectOptions = computed(
-    (): IFormSelectOption[] => accounts.value.map((acc) => ({
-      text: getAccountNameToDisplay(acc),
+  function prepareAccountsSelectOptions(withName = false): IFormSelectOption[] {
+    return accounts.value.map((acc) => ({
+      text: withName ? getAccountNameToDisplay(acc) : acc.address,
       value: acc.address,
       address: acc.address,
-    })),
-  );
+      listOnlyText: getAccountNameToDisplay(acc),
```

Comment:
In my opinion this is bad to have a possibility of 2 properties with the same values (when the `withName` is set to `true`.